openhgnn/output/DHNE/DHNE.md

```diff
@@ -0,0 +1,75 @@
+# DHNE[AAAI2018]
+
+Paper: [Structural Deep Embedding for Hyper-Networks](https://arxiv.org/abs/1711.10146)
+
+Code from author: https://github.com/tadpole/DHNE
+
+### How to Run
+
+Clone the OpenHGNN
+
+```
+# For hypergraph task
+python main.py -m DHNE -d drug -t hypergraph -g 0 --use_best_config
+```
+
+If you do not have gpu, set -gpu -1.
+
+Candidate dataset: 
+
+​	drug/GPS/MovieLens/wordnet
+
+### Dataset
+
+Supported dataset: drug, GPS, MovieLens, wordnet
+
+The details of datasets can be seen in the graph below:
+
+![image-20221013115422414](C:\Users\81480\AppData\Roaming\Typora\typora-user-images\image-20221013115422414.png)
```

Comment:
I think there will be an error in this line, as in "github", we do not have this path. You may change it to a table in Markdown like [this](https://github.com/BUPT-GAMMA/OpenHGNN/blob/main/README_EN.md#supported-models-with-specific-task).